Corona-Warn-App/src/main/res/layout/activity_main.xml

```diff
@@ -6,12 +6,33 @@
     android:layout_height="match_parent"
     tools:context="de.rki.coronawarnapp.ui.main.MainActivity">
 
-    <androidx.fragment.app.FragmentContainerView
-        android:id="@+id/nav_host_fragment"
-        android:name="androidx.navigation.fragment.NavHostFragment"
+    <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
We could also achieve this with a `LinearLayout`, do you think performance will be better?

Reply:
I don't think it will have a big impact on the performance.